### ISSUE TITLE
faunadb import error

### ISSUE THREAD

**Issue by step135** [cloudflare/wrangler#1696]
It worked in the past, not now. Using `import faunadb from 'faunadb'` is causing following error:

```
wrangler publish
✨  Built successfully, built project size is 44 KiB.
Error: Something went wrong! Status: 400 Bad Request, Details {
  "result": null,
  "success": false,
  "errors": [
    {
      "code": 10021,
      "message": "Uncaught TypeError: Cannot set property 'AbortSignal' of undefined\n  at line 15\n  at line 1 in n\n  at line 15\n  at line 1 in n\n  at line 13\n  at line 1 in n\n  at line 13\n  at line 1 in n\n  at line 13\n  at line 1 in n\n"
    }
  ],
  "messages": []
}

```

**Comment by xtuc** [comment]
did you try to polyfill AbortSignal?

**Comment by gahabeen** [comment]
@step135  Are you instanciating your client as suggested by Fauna for Cloudflare Workers?
`new faunadb.Client({ secret: "<key>", fetch: fetch.bind(globalThis) })`

**Comment by eidam** [comment]
Seems like related to https://github.com/fauna/faunadb-js/issues/375

**Comment by eidam** [comment]
this has been fixed by faunadb `4.0.2`

**Comment by Electroid** [comment]
The Workers runtime does not support `AbortSignal`, but we probably should.
